docs/docs/configuration.md

```diff
@@ -69,19 +69,32 @@ downloaded if you know the package's id and version. You can override this behav
 
 ## Enable package overwrites
 
-Normally, BaGetter will reject a package upload if the id and version are already taken. You can configure BaGetter
-to overwrite the already existing package by setting `AllowPackageOverwrites`:
+Normally, BaGetter will reject a package upload if the id and version are already taken. This is to maintain the [immutability of semantically versioned packages](https://learn.microsoft.com/en-us/azure/devops/artifacts/artifacts-key-concepts?view=azure-devops#immutability).
```

Comment:
FYI, if you remove the language code from learn.microsoft.com URL, it'll redirect the user to their preferred language:

```suggestion
Normally, BaGetter will reject a package upload if the id and version are already taken. This is to maintain the [immutability of semantically versioned packages](https://learn.microsoft.com/azure/devops/artifacts/artifacts-key-concepts?view=azure-devops#immutability).
```